tests/Unit/Evolution/Systems/Ccz4/Christoffel.py

```diff
@@ -20,3 +20,9 @@ def christoffel_second_kind(conformal_spatial_metric,
                    (np.einsum("jl,i", conformal_spatial_metric, field_p) +
                     np.einsum("il,j", conformal_spatial_metric, field_p) -
                     np.einsum("ij,l", conformal_spatial_metric, field_p)))))
+
+
+def contracted_conformal_christoffel_second_kind(
+        inverse_conformal_spatial_metric, conformal_christoffel_second_kind):
```

Comment:
Note that the CI failures are requesting formatting here and in `DerivChristoffel.py`, but:
- when I run `yapf` locally on both files, `yapf` finds nothing that needs to change
- when I manually make these changes the CI is requesting, it prevents me from `git commit`ing locally successfully because it doesn't accept the formatting that the CI is requesting

Let me know what I can do to resolve this discrepancy!

Reply:
are you using version 0.29.0 of yapf? If not that's likely the issue

Reply:
Whoops I edited my response before the page had refreshed with your response. I am using `0.31.0`. I'll ask Dr. Lovelace to update the version on ocean.